integration_tests/pages/prisons/incentiveGroups/viewIncentiveGroupsPage.ts

```diff
@@ -6,6 +6,11 @@ export default class ViewIncentiveGroupsPage extends Page {
     super('Hewell (HMP)')
   }
 
+  checkOnPage(): void {
```

Comment:
Extend the parent class method for each of these section `Page` classes to give a better check of being on the page that checks the relevant sub-nav section is active.